test/tap/LoggingEvent-test.js

```diff
@@ -94,6 +106,9 @@ test('LoggingEvent', (batch) => {
     t.equal(event2.lineNumber, undefined);
     t.equal(event2.columnNumber, undefined);
     t.equal(event2.callStack, undefined);
+    t.equal(event2.className, undefined);
```

Comment:
Forgot to include `t.equal(event2.functionName, undefined);`?

Reply:
Yeah, noticed it myself too. Fixed in [6ea3e43](https://github.com/log4js-node/log4js-node/pull/1316/commits/6ea3e43c62f2d075a335b87967e3b1c8a1b28067)